src/transformers/models/auto/configuration_auto.py

```diff
@@ -301,7 +301,10 @@
         ("owlvit", "OwlViTConfig"),
         ("paligemma", "PaliGemmaConfig"),
         ("parakeet_ctc", "ParakeetCTCConfig"),
+        ("parakeet_tdt", "ParakeetTDTConfig"),
         ("parakeet_encoder", "ParakeetEncoderConfig"),
+        ("parakeet_tdt_decoder", "ParakeetTDTDecoderConfig"),
+        ("parakeet_tdt_joint", "ParakeetTDTJointConfig"),
```

Comment:
Such mappings can be removed after removing `ParakeetTDTDecoderConfig` and `ParakeetTDTJointConfig`